multi-llm-control-room/backend/app/routers/mcp.py

```
@@ -0,0 +1,91 @@
+        return {"status": "shutdown", "server": server_name}
+
+    except Exception as e:
+        raise HTTPException(status_code=500, detail=str(e))
+
+
+@router.get("/tools/schemas")
+async def get_tool_schemas(server_names: List[str]):
+    """Get OpenAI-compatible tool schemas for specified servers"""
+    try:
+        schemas = mcp_manager.get_tool_schemas_for_model(server_names)
+        return {"schemas": schemas}
```

Ruff (0.14.8)

Consider moving this statement to an `else` block

(TRY300)